src/controller/java/AndroidDeviceControllerWrapper.cpp

```diff
@@ -398,6 +398,12 @@ void AndroidDeviceControllerWrapper::Shutdown()
 {
     VerifyOrReturn(mIsInitialized);
     getICDClientStorage()->Shutdown();
+    FabricIndex fabricIndex = Controller().GetFabricIndex();
+    FabricTable * fabricTable = DeviceControllerFactory::GetInstance().GetSystemState()->Fabrics();
+    if (fabricIndex != kUndefinedFabricIndex && fabricTable != nullptr)
+    {
+        fabricTable->Delete(fabricIndex);
+    }
     mController->Shutdown();
```

Comment:
Are you sure controller shutdown does not use the fabric table for anything?

Seems to me like the fabric table update should be _after_ controller shutdown.

Separately: this is an API change, right?  Will this break any existing clients?  Or does Android never start controllers on existing fabrics?